### PR TITLE
Print escalation policy summary

### PR DESCRIPTION
**Pull request by jamiehannaford**
We noticed some ambiguous output in https://github.com/github/sre-lifecycle/issues/324. This PR makes it more obvious which EP is which.

/cc @keithduncan @skymob 
/cc @github/sre-lifecycle

### CODE REVIEW

**Comment by skymob** [comment]
@jamiehannaford should we also update the README to reflect the format of the additional `pager me` output?

File: src/scripts/pagerduty.coffee [hunk 1]
```diff
@@ -430,12 +430,13 @@ module.exports = (robot) ->
             user = entry.user.summary
           startTime = moment(entry.start).tz(timezone).format()
           endTime = "?"
+          ep_summary = entry.escalation_policy.summary
```
Comment (skymob) [inline src/scripts/pagerduty.coffee]: Tiny nitpick: the variable format for the rest of the script using camel case. Should we honor that by making `ep_summary`  -> `epSummary`?

**Comment by jamiehannaford** [comment]
@skymob RFR again :)

**Comment by keithduncan** [comment]
I think there’s a submodule reference added in error here :bow:

**Comment by jamiehannaford** [comment]
Whoops! That should be fixed now

**Comment by jamiehannaford** [comment]
🚀

**Pull request merged**